lua/cmp_nvim_lsp_signature_help/init.lua

```diff
@@ -99,7 +99,7 @@ source._item = function(self, signature, parameter_index)
   parameter_index = (signature.activeParameter or parameter_index or 0) + 1
 
   -- @see https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#signatureHelp
-  if #parameters < parameter_index then
+  if #parameters < parameter_index or parameter_index == 0 then
```

Comment:
Well `parameter_index` will be `0` if `activeParameter == -1` (line 99 above) and this leads to empty items.

But how about `parameter_index < 1`? This would also catch any other negative value for `activeParameter`.

Reply:
Yes. I think it should be.
We should support other negative values.

Reply:
Changed.